core/network/fabric-interop-cc/contracts/interop/Makefile

```diff
@@ -4,3 +4,12 @@ test-manage-assets: assetexchange
 	go test manage_assets.go manage_assets_test.go main.go setup_test.go -v
 clean:
 	rm -rf assetexchange
+build-local:
```

Comment:
Can't we use the `go.mod.local` in this folder for a local build?

Reply:
That's what this `build-local` is using.

Reply:
OK, I must have missed that.